tests/test_wagon.py

```diff
@@ -299,44 +263,6 @@ def test_install_package_in_missing_venv(self):
         assert 'virtualenv {0} does not exist'.format(non_existing_venv) \
             in str(ex.value)
 
-    @pytest.mark.skipif(not wagon.IS_LINUX, reason='Irrelevant on non-Linux')
-    def test_get_platform_properties(self):
-        os_properties = wagon._get_os_properties()
-        assert os_properties == \
-            distro.linux_distribution(full_distribution_name=False)
-
-    @mock.patch('wagon.IS_DISTRO_INSTALLED', False)
-    @pytest.mark.skipif(not wagon.IS_LINUX, reason='Irrelevant on non-Linux')
-    def test_get_platform_properties_without_distro(self):
-        os_properties = wagon._get_os_properties()
-        assert os_properties == \
-            platform.linux_distribution(full_distribution_name=False)
-
-    def _test_get_env_bin_path(self):
-        tmp_env = wagon._make_virtualenv()
-        try:
-            env_bin_path = wagon._get_env_bin_path(tmp_env)
-            assert env_bin_path == virtualenv.path_locations(tmp_env)[3]
-        finally:
-            shutil.rmtree(tmp_env, ignore_errors=True)
-
-    def test_get_env_bin_path(self):
-        self._test_get_env_bin_path()
-
-    @mock.patch('wagon.IS_VIRTUALENV_INSTALLED', False)
-    def test_get_env_bin_path_without_virtualenv(self):
-        self._test_get_env_bin_path()
-
-    @mock.patch('wagon.IS_VIRTUALENV_INSTALLED', False)
-    def test_assert_virtualenv_is_not_installed(self):
```

Comment:
this, and the one below, mock out the only moving part, and check that the code is written the way we did